Dockerfile

```diff
@@ -51,3 +51,23 @@ EXPOSE 3000
 ENV PORT 3000
 
 CMD ["server.js"]
+
+# EDUCATIONS
+
+FROM node:14.17.0
```

Comment:
Currently, we're using node >= 20 since it's the latest LTS version or we can switch to >= 22 already to be more future proof.